app/controllers/journals_controller.rb

```diff
@@ -51,8 +48,12 @@ def journal_relations
   end
 
   def set_journal
-    @journal = Journal.find(params[:id])
-    authorize @journal
+    if Journal.exists?(params[:id])
```

Comment:
I Think you can use find_by_id, because that won't throw an exception if id is not found

```ruby
@journal = Journal.find_by_id(params[:id])
if @journal
  authorize @journal
else
  redirect_to ...
end
```